lib/rules/jsx-no-literals.js

```diff
@@ -44,6 +44,7 @@ module.exports = {
 
     function getValidation(node) {
       const standard = !/^[\s]+$/.test(node.value) &&
+          typeof node.value === 'string' &&
```

Comment:
hm, i'm kind of confused - what's the typeof `node.value` if it's an object, or a function? Can you add some test cases for that?